### ISSUE TITLE
Calibration .txt files

### ISSUE THREAD

**Issue by villanuevab** [kujason/avod#91]
Which of the calibration parameters in `calib/xxxxx.txt` are required for inference? 

`P0-P3` ???
`R0_rect` Required
`Tr_velo_to_cam` Required
`Tr_imu_to_velo` ???

**Comment by villanuevab** [comment]
It looks like `P2` parameters are required. I am still confused by why there are parameters implying a stereo setup (e.g., `KEY_STEREO_CALIB_P2` in `kitti_dataset.py`). We run training and inference with input from a single camera and single lidar only. Is this understanding correct?

**Comment by walzimmer** [comment]
Yes your are right. KITTI uses two stereo cameras (one monochrome and one RGB stereo camera) so that they have four cameras in total. AVOD only uses the left RGB camera and KEY_STEREO_CALIB_P2 is the projection matrix of that left RGB camera. For inference you need the projection matrix P2, the rectification matrix R0_rect and the transformation matrix Tr_velo_to_cam.

**Comment by villanuevab** [comment]
Upon closer inspection, the only calibration parameter in the feed_dict (model input) is: 
```
stereo_calib_p2 = calib_utils.read_calibration(self.calib_dir,
                                                           int(sample_name)).p2
```
See here: https://github.com/kujason/avod/blob/master/avod/datasets/kitti/kitti_dataset.py#L285-L287

Thus, it doesn't look like `R0_rect` or `Tr_velo_to_cam` are used at all. Can you share where you see that `R0_rect` or `Tr_velo_to_cam` are used for inference? 

Thank you for your time...

**Comment by yinhai86924** [comment]
> Yes your are right. KITTI uses two stereo cameras (one monochrome and one RGB stereo camera) so that they have four cameras in total. AVOD only uses the left RGB camera and KEY_STEREO_CALIB_P2 is the projection matrix of that left RGB camera. For inference you need the projection matrix P2, the rectification matrix R0_rect and the transformation matrix Tr_velo_to_cam.

why not use  the rectification matrix R2_rect instead of  the rectification matrix R0_rect ? beacouse it uses the left RGB camera,   R0_rect instand for the 0 gray camera ' rectification matrix R0_rect
